handlers/secrets_test.go

```diff
@@ -0,0 +1,130 @@
+package handlers
+
+import (
+	"testing"
+
+	"github.com/openfaas/faas/gateway/requests"
+	apiv1 "k8s.io/api/core/v1"
+	v1beta1 "k8s.io/api/extensions/v1beta1"
+)
+
+func Test_UpdateSecrets(t *testing.T) {
```

Comment:
This test name needs to be more descriptive so 6 months later we know what "Test Update Secrets" was trying to do.

Reply:
Also - I commented out the code change and this test still seemed to pass. I wasn't expecting that.

Reply:
If this is meant to be a (retro-fitted) regression test then it should also have a better name describing what is being tested - `Test_UpdateSecrets` doesn't give enough information to someone having to maintain this code in the future.